pkg/table/table_test.go

```diff
@@ -271,7 +271,7 @@ func TestAppendInstances(t *testing.T) {
 	}
 
 	addedInstanceIds = append(addedInstanceIds, *instances[0].InstanceId)
-	data, indexedOptions, _ = table.AppendInstances(data, indexedOptions, instances, addedInstanceIds)
+	data, indexedOptions, _, _ = table.AppendInstances(data, indexedOptions, instances, addedInstanceIds)
 	th.Equals(t, expectedData, data)
```

Comment:
This assertion is [failing in the unit tests:](https://github.com/GavinBurris42/aws-simple-ec2-cli/runs/7644691233?check_suite_focus=true#step:4:423)

```
=== RUN   TestAppendInstances
table_test.go:275:

	exp: [][]string{[]string{"1.", "Instance 2(i-67890)", "", ""}, []string{"2.", "Instance 3(i-54321)", "CreatedBy", "simple-ec2"}, []string{"", "", "CreatedTime", "just now"}, []string{"3.", "i-09876", "", ""}}

	got: [][]string{[]string{"i-09876", "", ""}}

--- FAIL: TestAppendInstances (0.00s)
```